.github/workflows/deploy-personal-dev-environment.yml

```diff
@@ -7,6 +7,14 @@ on:
         description: "The name of the dev environment you wish to deploy to e.g. abcd1234"
         required: true
         type: string
+      frontend_branch:
+        description: "The name of the git branch from the frontend which should be deployed"
+        default: "main"
+        type: string
```

Comment:
For now we'll just take raw strings as inputs, I wanna see if we can just prepopulate this list of choices from the branches available in that remote repo. But for now I think this will do